models/application.js

```diff
@@ -23,22 +23,17 @@ module.exports = function(sequelize, DataTypes) {
                 type: DataTypes.STRING,
                 allowNull: false
             },
-            application_title: {
-                type: DataTypes.STRING,
-                allowNull: false
-            },
             application_data: {
                 type: DataTypes.JSON,
                 allowNull: true
             },
             status: {
                 type: DataTypes.ENUM,
-                values: ['ineligible', 'eligible', 'pending', 'complete'],
-                allowNull: true
+                values: ['pending', 'complete'],
```

Comment:
This might involve a manual step when merging to `TEST` as I don't know if Sequelize is able to automatically migrate this kind of thing, just FYI.

Reply:
Easiest thing might be to rename the old column, manually make a new one with the right config, then merge this PR so it doesn't fail/complain? Though if it works locally then ignore me.

Reply:
Sorry should've called out that I removed the `freezeTableName` setting here so this actually creates a _new_ table called `applications` rather than the current `application`. I checked and the former doesn't exist on either the test or live databases so this would create a brand new table.

This is a good broader point though that we'll soon need a better strategy for migrating database schema changes, but for now this is essentially starting from scratch for this table.